babel.config.js

```diff
@@ -0,0 +1,18 @@
+// Used by Jest to transpile files before running test.
```

Comment:
This is needed because you dropped the use of `--experimental-vm-modules` right? If yes, why ?

Reply:
My understanding is that you have two choices:
- either migrate the package you want to test to ESM, and use `--experimental-vm-modules`
- or make sure that the ESM packages you depend on are transpiled to CJS

I chose the second option because it seems more atomic: migrating `activeui-migration` to pure ESM seems more out of scope of this PR.
This is also in line with what we did in the ActiveUI monorepo: first we made sure that `Jest` could still be ran in packages that they were not fully ESM yet (most of our packages), and then progressively we migreate each package to ESM, separately.